src/test/java/tutorlink/command/FindStudentCommandTest.java

```diff
@@ -0,0 +1,80 @@
+//@@author RCPilot1604
+package tutorlink.command;
+
+import java.util.HashMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import tutorlink.appstate.AppState;
+import tutorlink.exceptions.IllegalValueException;
+import tutorlink.exceptions.StudentNotFoundException;
+import tutorlink.result.CommandResult;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class FindStudentCommandTest {
+    private FindStudentCommand findCommand;
+    private AddStudentCommand addCommand;
+    private final AppState appState = new AppState();
+    private HashMap<String, String> arguments;
+    private CommandResult result;
+
+    @BeforeEach
+    void setup() {
+        findCommand = new FindStudentCommand();
+        addCommand = new AddStudentCommand();
+        arguments = new HashMap<>();
+        arguments.put("i/","A1234567X");
+        arguments.put("n/", "John");
+        result = addCommand.execute(appState, arguments);
+        assertNotNull(result);
+        arguments.put("i/","A2234567X");
+        arguments.put("n/", "Jon");
+        result = addCommand.execute(appState, arguments);
+        assertNotNull(result);
+    }
+
+    @Test
+    void execute_matric_one() {
```

Comment:
Test method names don't follow the recommended format` featureUnderTest_testScenario_expectedBehavior()`. Consider renaming.